src/pages/settings/Security/MergeAccounts/AccountValidatePage.tsx

```diff
@@ -92,21 +107,36 @@ function AccountValidatePage() {
 
     useFocusEffect(
         useCallback(() => {
-            if (!errorKey || !email) {
+            if (!errorPage || !email) {
                 return;
             }
-            return Navigation.navigate(ROUTES.SETTINGS_MERGE_ACCOUNTS_RESULT.getRoute(email, errorKey), {forceReplace: true});
-        }, [errorKey, email]),
+            return Navigation.navigate(ROUTES.SETTINGS_MERGE_ACCOUNTS_RESULT.getRoute(email, errorPage), {forceReplace: true});
+        }, [errorPage, email]),
     );
 
-    useFocusEffect(
-        useCallback(() => {
-            return () => {
-                clearMergeWithValidateCode();
-                clearGetValidateCodeForAccountMerge();
-            };
-        }, []),
-    );
+    useEffect(() => {
+        const unsubscribe = navigation.addListener('blur', () => {
+            clearGetValidateCodeForAccountMerge();
+            clearMergeWithValidateCode();
+        });
+
+        return unsubscribe;
+    }, [navigation]);
+
+    const validateCodeError = useMemo(() => {
```

Comment:
There's no slow code in this function so let's remove `useMemo` please.

Reply:
Done!